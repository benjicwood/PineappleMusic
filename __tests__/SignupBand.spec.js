/* global expect, it */
import 'react-native';
import React from 'react';
// import PineappleFront from '../index.android.js';
import SignupBand from '../views/signupBand/index.js';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <SignupBand />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
