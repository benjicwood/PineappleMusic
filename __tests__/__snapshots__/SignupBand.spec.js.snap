exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }>
  <Image
    resizeMode="cover"
    source={1}
    style={
      Array [
        Object {
          "flex": 1,
        },
        Object {
          "height": null,
          "paddingTop": 30,
          "width": null,
        },
      ]
    }>
    <View
      style={
        Object {
          "flex": 1,
        }
      }>
      <View
        style={
          Object {
            "backgroundColor": "transparent",
            "marginLeft": 10,
          }
        }>
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          hitSlop={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "height": 25,
              "opacity": 1,
              "width": 25,
            }
          }
          testID={undefined}>
          <Image
            resizeMode="contain"
            source={1}
            style={
              Object {
                "height": 25,
                "width": 25,
              }
            } />
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "transparent",
            "marginLeft": 25,
            "marginTop": 25,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontSize": 40,
            }
          }>
          Band Sign Up
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "flex": 3,
          "marginTop": 50,
        }
      }>
      <View
        style={
          Object {
            "borderBottomColor": "#CCC",
            "borderColor": "transparent",
            "borderWidth": 1,
            "flexDirection": "row",
            "height": 75,
          }
        }>
        <View
          style={
            Object {
              "alignItems": "center",
              "justifyContent": "center",
              "paddingHorizontal": 15,
            }
          }>
          <Image
            resizeMode="contain"
            source={1}
            style={
              Object {
                "height": 30,
                "width": 30,
              }
            } />
        </View>
        <TextInput
          placeholder="Band Name"
          placeholderTextColor="#FFF"
          style={
            Array [
              Object {
                "flex": 1,
                "fontSize": 20,
              },
              Object {
                "color": "#FFF",
              },
            ]
          }
          underlineColorAndroid="transparent" />
      </View>
      <View
        style={
          Object {
            "borderBottomColor": "#CCC",
            "borderColor": "transparent",
            "borderWidth": 1,
            "flexDirection": "row",
            "height": 75,
          }
        }>
        <View
          style={
            Object {
              "alignItems": "center",
              "justifyContent": "center",
              "paddingHorizontal": 15,
            }
          }>
          <Image
            resizeMode="contain"
            source={1}
            style={
              Object {
                "height": 30,
                "width": 30,
              }
            } />
        </View>
        <TextInput
          placeholder="Email"
          placeholderTextColor="#FFF"
          style={
            Array [
              Object {
                "flex": 1,
                "fontSize": 20,
              },
              Object {
                "color": "#FFF",
              },
            ]
          } />
      </View>
      <View
        style={
          Object {
            "borderBottomColor": "#CCC",
            "borderColor": "transparent",
            "borderWidth": 1,
            "flexDirection": "row",
            "height": 75,
          }
        }>
        <View
          style={
            Object {
              "alignItems": "center",
              "justifyContent": "center",
              "paddingHorizontal": 15,
            }
          }>
          <Image
            resizeMode="contain"
            source={1}
            style={
              Object {
                "height": 30,
                "width": 30,
              }
            } />
        </View>
        <TextInput
          placeholder="Password"
          placeholderTextColor="#FFF"
          secureTextEntry={true}
          style={
            Array [
              Object {
                "flex": 1,
                "fontSize": 20,
              },
              Object {
                "color": "#FFF",
              },
            ]
          } />
      </View>
      <View
        style={
          Object {
            "borderBottomColor": "#CCC",
            "borderColor": "transparent",
            "borderWidth": 1,
            "flexDirection": "row",
            "height": 75,
          }
        }>
        <View
          style={
            Object {
              "alignItems": "center",
              "justifyContent": "center",
              "paddingHorizontal": 15,
            }
          }>
          <Image
            resizeMode="contain"
            source={1}
            style={
              Object {
                "height": 30,
                "width": 30,
              }
            } />
        </View>
        <TextInput
          placeholder="Genre"
          placeholderTextColor="#FFF"
          style={
            Array [
              Object {
                "flex": 1,
                "fontSize": 20,
              },
              Object {
                "color": "#FFF",
              },
            ]
          }
          underlineColorAndroid="transparent" />
      </View>
    </View>
    <View
      style={
        Object {
          "flex": 1,
        }
      }>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
        testID={undefined}>
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FF3366",
              "justifyContent": "center",
              "marginBottom": 15,
              "paddingVertical": 25,
            }
          }>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#FFF",
              }
            }>
            Create a Pineapple
          </Text>
        </View>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
        testID={undefined}>
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "justifyContent": "center",
            }
          }>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#D8D8D8",
              }
            }>
            Already have an account?
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "#FFF",
                }
              }>
               Sign In
            </Text>
          </Text>
        </View>
      </View>
    </View>
  </Image>
</View>
`;
